### PR TITLE
oficial_devices: Add mido

### PR DESCRIPTION
**Pull request by NRanjan-17**

### CODE REVIEW

**Comment by NRanjan-17** [comment]
@Subinsmani @Darkstar085

File: devices.json [hunk 1]
```diff
@@ -253,11 +253,11 @@
          "camera": "13Mpx + 5Mpx",
          "cpu": "Snapdragon 625",
          "display": "IPS 5.5\" 1080p",
-         "ram": "2/3/4GB LPDDR4X"
+         "ram": "2/3/4GB LPDDR3"
       },
       "supported_bases": [
          {
-            "name": "ten",
```
Comment (Subinsmani) [inline devices.json]: Why removing ten base?

Comment (NRanjan-17) [reply devices.json]: ok sir

Comment (NRanjan-17) [reply devices.json]: fixing it

**Pull request closed without merging**